### DISCUSSION TITLE
Microprofile 5 Java baseline

### DISCUSSION THREAD

**Discussion by anbusampath** [eclipse/microprofile#230] [General]
I was reading Microprofile 5 release candidate doc Java SE section - https://download.eclipse.org/microprofile/microprofile-5.0-RC1/microprofile-spec-5.0-RC1.html#javase

If Jakarta EE 9.1 required Java 11,  can Microprofile 5 also mandate Java 11 as baseline?

**Comment by Emily-Jiang** [comment]
Jakarta EE 9.1 does not preclude Java 8 but asks for Java 11 support. MicroProfile 5 allows Java 8+. I don't see why we should not mandate Java 11 as the baseline. Many runtimes such as Open Liberty supports MP 5 with Java 11.
